CHARTER.md

```diff
@@ -0,0 +1,73 @@
+ - Formation of technical **working groups**.
+ - **Representation** in technical collaborations with external communities.
+
+### 1.2: Out-of-Scope
+
+Section Intentionally Left Blank
+
+## Section 2: Relationship with OpenJS Foundation CPC.
+
+Technical leadership for the projects within the [OpenJS Foundation](https://openjsf.org/) is delegated to the projects through their project charters by the [OpenJS Foundation Cross-Project Council](https://openjsf.org/about/governance/) (CPC). In the case of the `vis.gl` project, it is delegated to the [Technical Steering Committee](https://github.com/visgl/tsc/blob/master/README.md#technical-steering-committee) (the “TSC”). The OpenJS Foundation's business leadership is the Board of Directors (the “Board”).
+
+This `vis.gl` Charter reflects a carefully constructed balanced role for the TSC and the CPC in the governance of the OpenJS Foundation. The charter amendment process is for the TSC to propose changes using simple majority of the full TSC, the proposed changes being subject to review and approval by the CPC. The CPC may additionally make amendments to the project charter at any time, though the CPC will not interfere with day-to-day discussions, votes, or meetings of the TSC.
+
+### 2.1 Other Formal Project Relationships
+
+Section Intentionally Left Blank
+
+## Section 3: `vis.gl`'s TSC Governing Body
+
+`vis.gl` is governed by its [Technical Steering Committee](https://github.com/visgl/tsc/blob/master/README.md#technical-steering-committee).
+
+## Section 4: Roles & Responsibilities 
+
+The roles and responsibilities of `vis.gl`'s TSC are described in [GOVERNANCE.md](./GOVERNANCE.md).
+
+### Section 4.1 Project Operations & Management
+
+Section Intentionally Left Blank
+
+### Section 4.2: Decision-making, Voting, and/or Elections
+
+The vis.gl TSC operates on a [consensus-seeking basis](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making). Most decisions are expected to be resolved through open discussion and agreement. However, when consensus cannot be reached, any TSC member may call for a vote.
+
+ - **Eligibility:** All steering committee members are voting members and may participate in any vote.
+
+ - **Quorum:** A quorum is established when at least 50% of steering committee members members are present. The steering committee members may meet without quorum, but may not make binding decisions.
+
+ - **Votes During Meetings:** When quorum is met, decisions require a simple majority of the steering committee members present.
+
+ - **Electronic Votes:** Decisions made asynchronously (e.g., over email or GitHub) require a simple majority of all steering committee members.
+
+ - **Escalation:** If a vote results in a tie or fails to resolve a dispute, any TSC member may refer the matter to the OpenJS Foundation Board for mediation or guidance.
```

Comment:
We'd want the TSC chair (not any TSC member) to raise this with the CPC (not the Board).